LazyAlchemyLearner.lua

```diff
@@ -369,5 +385,10 @@ SLASH_COMMANDS['/lazylearn'] = genericSlashCommand
 SLASH_COMMANDS["/learnalchemytraits"] = function(arg)
     -- Call the queueLearningAlchemy function with the parsed argument
     queueLearningAlchemy(arg == "all")
+    if arg == 'all' then
+		d("DLC + Base game alchemy traits queued")
+	else
+		d("Base game alchemy traits queued. Type /lazylearn all to also learn DLC traits")
```

Comment:
Might be a copy paste mistake. 
You might have intended to say "Type /learnalchemytraits all to also learn..." in this case.

Reply:
It wasn't a typo, but you're right that it should probably be the same command used. I'd wanted to funnel people to the more comprehensive slash command, but its probably better to just have them use the one they're already using